Makefile

```diff
@@ -57,11 +57,11 @@ SRCS := About.cpp \
 
 #	Specify the resource definition files to use. Full or relative paths can be
 #	used.
-RDEFS = 
+RDEFS = Radio.rdef
 
 #	Specify the resource files to use. Full or relative paths can be used.
 #	Both RDEFS and RSRCS can be utilized in the same Makefile.
-RSRCS = Radio.rsrc
```

Comment:
If you are removing the RSRC, please delete it too.

Reply:
Done. Thanks!